dvc/repo/add.py

```diff
@@ -50,11 +51,39 @@ def get_or_create_stage(
     to_remote: bool = False,
     force: bool = False,
 ) -> StageInfo:
+    """
+    Adds a new tracked file or update an existing one.
+
+    Used in the context of dvc-add.
+
+    Args:
+        target : an expression that resolves to a ...
+        out : if specified, what does this to?
+        to_remote : if True, what does this to?
+        force : what does this to?
```

Comment:
See https://dvc.org/doc/command-reference/add#-f

Reply:
Can I add explicit documentation here? Alternatively, can I include docs that point to references on where these concepts are enumerated explicitly? Or would it be better to have code with no docs here?

Reply:
I don't have a strong opinion but maybe @skshetry can weigh in